packages/parachains-impl/amplitude/hooks/index.ts

```diff
@@ -0,0 +1,13 @@
+export * from '../../bifrost/hooks/useAddLiquidityStandardReview'
+export * from '../../bifrost/hooks/useClaimFarmingRewardsReview'
+export * from '../../bifrost/hooks/usePairTotalSupply'
+export * from '../../bifrost/hooks/useRemoveLiquidityStandardReview'
+export * from '../../bifrost/hooks/useStablePools'
+export * from '../../bifrost/hooks/useStakeLiquidityStandardReview'
+export * from '../../bifrost/hooks/useSwapReview'
+export * from '../../bifrost/hooks/useTotalSupply'
```

Comment:
This is just reexport the bifrost impl, If there is a break upgrade on bifrost, it maybe not work on your chain. So It's better you can have your own impl.

Reply:
Right, that's a good point. I changed it so that the hooks for Amplitude don't reference the ones of Bifrost anymore.
However, I kept the [components](https://github.com/pendulum-chain/zenlink-interface/blob/55125c20b198bd58ecb68aec0fb4ca7927b3584c/packages/parachains-impl/amplitude/components/index.ts) as-is meaning that the components for Amplitude will still be the same as for Bifrost. I decided not to change that to reduce maintenance overhead for these components. Is that okay @haike0513 or would you also suggest to duplicate those?